src/Raven.Client/Documents/DocumentStore.cs

```diff
@@ -426,7 +426,7 @@ public override MaintenanceOperationExecutor Maintenance
             get
             {
                 AssertInitialized();
-                return _maintenanceOperationExecutor ?? (_maintenanceOperationExecutor = new MaintenanceOperationExecutor(this));
+                return MaintenanceOperationExecutor ?? (MaintenanceOperationExecutor = new MaintenanceOperationExecutor(this));
```

Comment:
I guess we can rename it back after adding this:
https://github.com/ravendb/ravendb/pull/12863/files#r825432960